### DISCUSSION TITLE
Trigger only when scrolling down?

### DISCUSSION THREAD

**Discussion by adamreisnz** [Akryum/vue-observe-visibility#227] [Q&A]
I was wondering if there's a way to trigger the callback only when a user is scrolling down?
Looking around the options I couldn't find anything obvious, some tips would be much appreciated.

**Comment by adamreisnz** [comment]
Found some references here https://stackoverflow.com/questions/46478202/how-do-i-know-the-intersectionobserver-scroll-direction, but wonder if additional properties like this could get passed to our callback by this plugin.

**Comment by Akryum** [comment]
You have access to the `entry` as the second argument of the callback.
